src/PluginRegistrationDialog.h

```diff
@@ -2,418 +2,24 @@
 
   Audacity: A Digital Audio Editor
 
-  PluginManager.cpp
+  PluginRegistrationDialog.h
```

Comment:
This new header file contains pieces of what used to be in PluginManager.cpp.

Added lines are necessary include guards and include directives and forward declarations, and also class ItemData which was moved out of PluginManager.cpp and made a nested class.

Otherwise it is easy to see that the definition of class PluginRegistrationDialog is unchanged.

Everything else in PluginManager.cpp that was not moved here is deleted from the copy.